cmd/rook/ceph/osd.go

```diff
@@ -241,8 +241,13 @@ func commonOSDInit(cmd *cobra.Command) {
 func getLocation(clientset kubernetes.Interface) (string, error) {
 	// get the value the operator instructed to use as the host name in the CRUSH map
 	hostNameLabel := os.Getenv("ROOK_CRUSHMAP_HOSTNAME")
+	rootLabel := os.Getenv("ROOK_CRUSHMAP_ROOT")
```

Comment:
Can't we always populate this with a value instead? So if `osdProps.crushRoot` is empty then we set `default`. I see multiple places use the same check and I believe we can solve this in a single place, when we set the `ROOK_CRUSHMAP_ROOT` basically.